apps/web/scripts/generate-cms-dictionaries.ts

```
@@ -0,0 +1,65 @@
+async function main() {
+  AVAILABLE_LANGS.map(async (lang) => {
+    let categories = (await getArticleSections(lang as Lang)).map(
+      (section) => section.slug
+    );
+
+    categories.map(async (category) => {
+      const articles = (
+        await getBlogCategoryLandingData(lang as Lang, category)
+      ).sections.map((section) => section.articles);
+
+      const slugs = articles.map((art) => art.map((a) => a.slug)).flat();
```

Biome

[error] The call chain .map().flat() can be replaced with a single .flatMap() call.

Safe fix: Replace the chain with .flatMap().


(lint/complexity/useFlatMap)